google/oauth2/reauth.py

```diff
@@ -275,8 +275,10 @@ def refresh_grant(
     client_secret,
     scopes=None,
     rapt_token=None,
+    enable_reauth_refresh=False,
 ):
-    """Implements the reauthentication flow.
+    """Implements the reauthentication flow. This method is intended for gcloud
```

Comment:
This method is used no matter if reauth is enabled, right? enable_reauth_refresh should be used only by gcloud.

Reply:
moved comments to `enable_reauth_refresh`.